src/nn/wasm/PreparedModel.js

```diff
@@ -1,19 +1,12 @@
-import getNNOpsInstance from './NNOps'
-import { OperationCode, OperandCode, PaddingCode, PreferenceCode, FuseCode, OperandLifetime } from '../Enums'
-import * as utils from '../utils'
-import { product } from '../utils';
+import { Mutex } from '../utils';
+import { getPreparedModelWorker } from './WorkerIPC';
+
+let worker = getPreparedModelWorker();
```

Comment:
Are you able to make the use of worker as an option?

Reply:
Not yet. I will find an appropriate place to add this option